src/Nancy/ArrayCache.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Nancy
+{
+    public class ArrayCache

```

Comment:
You know better than to try and sneak undocumented classes below my radar :stuck_out_tongue_closed_eyes: